action.yml

```diff
@@ -0,0 +1,9 @@
+# action.yml
+name: 'NLU Hyperopt'
+description: 'Find optimal hyperparameters for NLU model'
+outputs:
+  best-intent-f1: # id of output
+    description: 'best intent macro averaged f1 score found'
```

Comment:
does this actually work? shouldn't this run docker-compose?

Reply:
it appears docker-compose is not supported in action.yml? Or I'm doing it wrong.
```##[error]System.ArgumentOutOfRangeException: Specified argument was out of the range of valid values. (Parameter ''using: docker-compose' is not supported, use 'docker' or 'node12' instead.')
```

Reply:
@wochinge Ok! As you can see from the long list of tiny commits (...couldn't test if it worked otherwise...) it took a while but it's working now. Mostly had to figure out how people could overwrite `space.py` from their workflow file.